packages/gatsby/cache-dir/head/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Helmet } from "react-helmet-async"
+import { useLocation } from "@reach/router"
+
+export const Head = props => {
+  const location = useLocation()
+
+  // This supports passing a gatsby-plugin-image object to the image prop, e.g.
+  // image={getImage(data.metaImage)}
+  const image = props.image?.images?.fallback?.src || props.image
```

Comment:
In the case of `gatsby-plugin-image` we also get back the image size:

![image](https://user-images.githubusercontent.com/16143594/105977175-58b95f00-6091-11eb-9122-fb3c8add56b6.png)

So we could also add:

og:image:width - The number of pixels wide.
og:image:height - The number of pixels high.